test/README.md

```diff
@@ -150,9 +173,13 @@ In-test variables
                          Optionally a MODEL suffix can used for further filtering, e.g.
                          win32 win64 linux32 linux64 osx32 osx64 freebsd32 freebsd64
 
-Relevant environment variables
+Makefile Environment variables
 ------------------------------
 
+The Makefile uses environment variables to store test settings and as a way to pass these settings to the test wrapper tool `d_do_test`.
+
+> Note: These variables are available in any Bash test via ${VAR_NAME}
```

Comment:
Note: Technically there's no difference between `$VAR_NAME` and `${VAR_NAME}` (for strings at least) except when that `${foo}${bar}` doesn't work with `$foo$bar`.
It's one of these legacy things that no one dares to deprecate because it would affect millions of millions of lines of code.

https://stackoverflow.com/questions/18135451/what-is-the-difference-between-var-var-and-var-in-the-bash-shell

So maybe we should just say: "These variables are available in the Bash tests too" ?